addons/binding/org.openhab.binding.seneye/src/main/java/org/openhab/binding/seneye/internal/SeneyeHandlerFactory.java

```diff
@@ -17,14 +17,18 @@
 import org.eclipse.smarthome.core.thing.ThingTypeUID;
 import org.eclipse.smarthome.core.thing.binding.BaseThingHandlerFactory;
 import org.eclipse.smarthome.core.thing.binding.ThingHandler;
+import org.eclipse.smarthome.core.thing.binding.ThingHandlerFactory;
 import org.openhab.binding.seneye.handler.SeneyeHandler;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.ConfigurationPolicy;
 
 /**
  * The {@link SeneyeHandlerFactory} is responsible for creating things and thing
  * handlers.
  *
  * @author Niko Tanghe - Initial contribution
  */
+@Component(service = ThingHandlerFactory.class, immediate = true, configurationPid = "binding.seneye", configurationPolicy = ConfigurationPolicy.OPTIONAL)

```

Comment:
Without a valid configuration you binding will not work anyway, right? So why do you set the `configurationPolicy` to optional here?

Reply:
ok, set to required

Reply:
Actually I just read [here](http://felix.apache.org/documentation/subprojects/apache-felix-maven-scr-plugin/scr-annotations.html#component) that `OPTIONAL` is the default and all our other bindings do not set it, so you can remove it, i.e. keep it optional. i am sorry.

I think the reason why its optional is that it will be started so you actually see it for example in paperUI and then you can configure it.